### DISCUSSION TITLE
Automations following v9.0.0 update

### DISCUSSION THREAD

**Discussion by AGLAISTER** [BottlecapDave/HomeAssistant-OctopusEnergy#552] [Q&A]
I have an automation that activates a fingerbot to trigger a 15 or 30 minute on my iBoost when my agile rates are 0p or less.

With the upgrade to v9.0.0. I believe the old sensors have been replaced by events.

In my automation do I need to replace my condition on current rate money with the new event or will it still work fine with the sensor?

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/141528175/e78fa786-07a9-4e22-b6cc-42798bd46086)

**Comment by BottlecapDave** [comment]
If you are using the current rate sensor _as is_, you shouldn't need to make any changes. You only need to make changes if you are using some of the attributes from the sensor (or something you're using relies on the attributes)

**Comment by AGLAISTER** [comment]
Thanks that's great news

Alan
